README.adoc

```diff
@@ -0,0 +1,274 @@
+....
+* To configure a
+https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/components/centos/centos-k8s-singlenode-setup/Vagrantfile[Kubernetes
+specific container development environment] use:
++
+....
+$ curl -sL https://raw.githubusercontent.com/projectatomic/adb-atomic-developer-bundle/master/components/centos/centos-k8s-singlenode-setup/Vagrantfile > Vagrantfile
+....
+* To configure an
+https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/components/centos/centos-openshift-setup/Vagrantfile[OpenShift
+Origin specific container development environment] use:
++
+....
+$ curl -sL https://raw.githubusercontent.com/projectatomic/adb-atomic-developer-bundle/master/components/centos/centos-openshift-setup/Vagrantfile > Vagrantfile
+....
+* To configure an
+https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/components/centos/centos-mesos-marathon-singlenode-setup/Vagrantfile[Apache
+Mesos Marathon specific container development environment] use:
++
+....
+$ curl -sL https://raw.githubusercontent.com/projectatomic/adb-atomic-developer-bundle/master/components/centos/centos-mesos-marathon-singlenode-setup/Vagrantfile > Vagrantfile
+....
+
+. Start ADB:
++
+----
+vagrant up
+----
++
+This will download ADB and set it up to work with the provider of
+choice, for use with host-based tools or via `vagrant ssh`. You may wish
+to review the link:docs/using.adoc[Using Atomic Developer Bundle]
+documentation before starting ADB, especially if you are using
+host-based tools.
+
+[[included-features]]
+== Which features are included?
+
+Currently, the box provides the following:
+
+* Docker support to unsupported platforms (i.e. Microsoft Windows, Mac
+OS X, etc.)
+* Kubernetes orchestration for local testing of applications
+* Application definition using the Nulecule specification
+
+Additional goals, objectives and work in progress can be found on the
+Project Atomic https://trello.com/b/j1rEolFe/container-tools[trello
+board].
+
+[[deliverables-and-downloading]]
+== What are the deliverables and where are they delivered?
+
+ADB is delivered as a Vagrant box for various (currently libvirt and
+VirtualBox) providers. The boxes are built using the CentOS powered
+https://wiki.centos.org/HowTos/CommunityBuildSystem[Community Build
+System]. Boxes are delivered via
+https://atlas.hashicorp.com/boxes/search[Hashicorp's Atlas] and are
+available at
+http://cloud.centos.org/centos/7/vagrant/x86_64/images/[cloud.centos.org].
+These boxes differ from existing Vagrant boxes for CentOS as they have
+specific build requirements that are not enabled in those boxes.
+
+[[documentation]]
+== Documentation
+
+* link:docs/installing.adoc[Installing ADB]
+* link:docs/using.adoc[How to use ADB]
+** link:docs/cockpit.adoc[Using Cockpit with ADB]
+* link:docs/updating.adoc[Updating ADB]
+* link:docs/building.adoc[Building the Vagrant box] for Developers
+
```

Comment:
Yes for now this is good, but ultimately they'll all need to be changed to the published doc URLs :)